libfreerdp/core/transport.c

```diff
@@ -1075,14 +1223,14 @@ int transport_check_fds(rdpTransport* transport)
 			return -1;
 		}
 
-		now = GetTickCount64();
+		//now = GetTickCount64();
 	}
 
-	if (now >= dueDate)
+/*	if (now >= dueDate)
```

Comment:
This can not be removed without a replacement, it is here for a reason (connection timeout)